Poppy updates - Users, Consortia, Bulk Edit, Lists, Settings > Consortium manager, and Settings > Users

c-tobias opened the pull request:
The Users document for Poppy.

c-tobias committed:
Update _index.md

Editing Users for Poppy. Still in progress but wanted to save changes made so far.

c-tobias committed:
Update _index.md

Users - Poppy

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/folio-org/docs?pullRequest=290) <br/>All committers have signed the CLA.

c-tobias committed:
Update _index.md

Poppy updates for Bulk Edit.

c-tobias committed:
Create _index.md

Consortia manager documentation for Poppy.

c-tobias committed:
Create Lists

c-tobias committed:
Update Settings_users.md

Poppy updates for Settings > Users. (DOCS-346)

c-tobias committed:
Delete content/en/docs/Lists

Mistake...trying to create a new folder for Lists and created a document instead. Okay to delete.

c-tobias committed:
Create _index.md

Corrected typo in directory (folder) name.

c-tobias committed:
Delete content/en/docs/consortia directory

Corrected typo in directory name. No longer using this directory.

c-tobias committed:
Create _index.md

New directory and file for Settings > Consortium manager.

c-tobias committed:
Create _index.md

New directory and file for Settings_consortium-manager

c-tobias committed:
Create _index.md

New directory and file for Lists.

julianladisch commented on content/en/docs/Lists/_index.md:
```diff
@@ -0,0 +1,325 @@
+linkTitle: "Lists"
+date: 2023-12-14        
+weight: 
+—
```
Three hyphens `---` end the YAML front matter.

julianladisch commented on content/en/docs/Consortia/_index.md:
```diff
@@ -0,0 +1,562 @@
+linkTitle: "Consortium manager"
+date: 2023-12-13        
+weight[a]: 
+—
```
Three hyphens `---` end the YAML front matter.

sekjal commented:
I'll update the hyphens after merge
